Makefile

```diff
@@ -271,11 +271,15 @@ $(CONTROLLER_GEN): $(LOCALBIN)
 
 .PHONY: envtest
 envtest: $(ENVTEST) ## Download envtest-setup locally if necessary.
-$(ENVTEST): $(LOCALBIN)
+$(ENVTEST): $(LOCALBIN) helm
```

Comment:
envtest doesn't depend on helm, maybe helm could just be a prerequisite for test-mk test-mk-v test-mk-do test-mk-do-v test-mk-do-fast test-mk-do-fast-v

Reply:
+1